src/ngx_http_lua_socket_tcp.c

```diff
@@ -782,6 +786,29 @@ ngx_http_lua_socket_tcp_connect(lua_State *L)
     return lua_yield(L, 0);
 }
 
```

Comment:
Style: we use 2 blank lines to separate successive C function definitions. See other similar places for examples.